scripts/setup/debians/build_install_debians.sh

```diff
@@ -22,7 +22,7 @@
 set -e
 
 debian_loc=$(dirname "$(readlink -f "$0")")
-dist=$(grep -oP "(?<=VERSION_CODENAME=).*" /etc/os-release)
+dist=$(. /etc/os-release && echo $UBUNTU_CODENAME)
```

Comment:
Do we still need this file with the build_debians.sh?

Reply:
I think I didn't test it (build_debians.sh) in the context of users so I was planning to leave them both for a while and take care of that on a future PR. We would also need to update the docs. For users, I think it would be build_debians.sh --install-with-deps.
Do you think we should replace it already?